hallux/proposals/python_proposal.py

```diff
@@ -43,7 +43,7 @@ def find_target_function(self, parsed_ast, issue_line):
     def count_code_offset(self):
         for line in self.issue_lines:
             lsline = line.lstrip(" ")
-            if len(lsline) > 0:
+            if len(lsline) > 0 and not (lsline == "\n" or lsline == "\r\n"):
```

Comment:
Do you agree to this change? This might be a reason why we have some empty lines in unit-tests, where others have "\n"..